charts/pvc-autoresizer/Chart.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
```

Comment:
TopoLVM has `home` and `sources` entries, is it OK not to have them in pvc-autoresizer?

Reply:
Thanks, I added:
https://github.com/topolvm/pvc-autoresizer/pull/54/commits/44ee24e0bd33f9e2462f476720b4cda46ca73260